pkg/framework/plugins/nodeutilization/nodeutilization.go

```diff
@@ -310,7 +310,7 @@ func evictPods(
 			}
 
 			if preEvictionFilterWithOptions(pod) {
-				if podEvictor.Evict(ctx, pod, evictions.EvictOptions{}) {
+				if podEvictor.Evict(ctx, pod, evictions.EvictOptions{StrategyName: "NodeUtilization"}) {
```

Comment:
There is no strategy called `NodeUtilization`. This is a shared function between LowNodeUtilization and HighNodeUtilization. Is it justified to pass in the plugin name? We could also have a common interface between the two plugins and make this a method a receiver on the interface